runtime/doc/nvim-extra-keymaps.txt

```diff
@@ -0,0 +1,36 @@
+*nvim-extra-keymaps.txt*    For nvim.                         {Nvim}
```

Comment:
`nvim` should be capitalized, and the dashes should be converted to underscored, both here, and in the actual file name. This is for consistency with all the other docs.